docs/source/conf.py

```diff
@@ -38,7 +38,7 @@
     # Toc options
     "collapse_navigation": False,
     "sticky_navigation": True,
-    "navigation_depth": 3,
+    "navigation_depth": 4,
```

Comment:
In contrast to above, I think 4 is fine here. I think it's only used for the sidebar (?) which uses progressive disclosure.